TestRailAutomationTest/Model/User.cs

```diff
@@ -0,0 +1,14 @@
+namespace TestRailAutomationTest.Model;
+
+public class User
+{
+    public string Email { get; }
+    
+    public string Password { get; }
+
+    public User(string email, string password)
+    {
+        Email = email;
+        Password = password;
+    }
```

Comment:
There is no need to specify constructors 

We can use C# object initialization 
https://www.tutorialsteacher.com/csharp/csharp-object-initializer#:~:text=C%23%20%2D%20Object%20Initializer%20Syntax,object%20without%20invoking%20a%20constructor.